src/genericSchemeRegistry/schemes/BalancerPoolManager.json

```diff
@@ -0,0 +1,341 @@
+        },
+        {
+          "decimals": 18,
+          "label": "Start block",
+          "name": "Start update from block",
+          "unit": "unit",
+          "placeholder": "Weight (199199)"
+        },
+        {
+          "decimals": 18,
+          "label": "End block",
+          "name": "End update at block",
+          "unit": "unit",
+          "placeholder": "Weight (399199)"
+        }
+      ],
+      "abi": {
+      "constant": false,
+      "inputs": [
+              {
+                "internalType": "uint256[]",
+                "name": "newWeights",
+                "type": "uint256[]"
+              },
+              {
+                "internalType": "uint256",
+                "name": "startBlock",
+                "type": "uint256"
+              },
+              {
+                "internalType": "uint256",
+                "name": "endBlock",
+                "type": "uint256"
+              }
+            ],
+            "name": "updateWeightsGradually",
+            "outputs": [],
+            "payable": false,
+            "stateMutability": "nonpayable",
+            "type": "function"
+            }
+    },
+    {
+      "id": "joinPool",
+      "label": "Join pool",
+      "description": "Joins the pool by adding more tokens",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "decimals": 18,
+          "label": "Pool Amount Out",
+          "name": "Number of pool tokens to receive",
+          "unit": "uint",
+          "placeholder": "Number (1000)"
+        },
+        {
+          "label": "Max Amounts In",
+          "name": "Max amount of asset tokens to spend",
+          "placeholder": "Amounts (10)"
+        }
+      ],
+      "abi": {
+          "constant": false,
+          "inputs": [
+            {
+              "internalType": "uint256",
+              "name": "poolAmountOut",
+              "type": "uint256"
+            },
+            {
+              "internalType": "uint256[]",
+              "name": "maxAmountsIn",
+              "type": "uint256[]"
+            }
+          ],
+          "name": "joinPool",
+          "outputs": [],
+          "payable": false,
+          "stateMutability": "nonpayable",
+          "type": "function"
+        }
+      },
+    {
+      "id": "exitPool",
+      "label": "Exit pool",
+      "description": "Exits the pool by redeeming the tokens",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "decimals": 18,
+          "label": "Pool Amount In",
+          "name": "Number of pool tokens to redeem",
+          "placeholder": "Number (1000)"
+        },
+        {
+          "label": "Min Amounts Out",
+          "name": "Minimum amount of asset tokens to receive",
+          "placeholder": "Amounts (10)"
```

Comment:
Supposed to be an array

Reply:
On the alchemy front-end there is a separate field for every element of the array